core/src/main/java/org/arquillian/smart/testing/logger/DefaultLoggerFactory.java

```diff
@@ -0,0 +1,104 @@
+package org.arquillian.smart.testing.logger;
+
+import org.arquillian.smart.testing.configuration.Configuration;
+
+import static java.lang.String.format;
+
+public class DefaultLoggerFactory implements LoggerFactory {
+
+    private boolean isDebugLogLevel = false;
```

Comment:
Do we really need to define default value? If you haven't define it will take `false` as default

Reply:
From my point of view, it's kind of transparency. But you are right - can be removed.